internal/security/proxy/main.go

```diff
@@ -75,7 +74,6 @@ func Main(ctx context.Context, cancel context.CancelFunc, _ *mux.Router, _ chan<
 		startupTimer,
 		dic,
 		[]interfaces.BootstrapHandler{
-			handlers.SecureProviderBootstrapHandler,
```

Comment:
Thought I had posted this comment, but don't see it so here goes again...

Also need to remove SecretStore configuration from TOML & structs.